src/components/Camera/StandardCamera.js

```diff
@@ -224,8 +251,43 @@ const StandardCamera = ( ): Node => {
         </View>
       </View>
       <Snackbar visible={showAlert} onDismiss={() => setShowAlert( false )}>
-        {t( "You-can-only-upload-20-media" )}
+        {t( "You-can-only-upload-20-media " )}
       </Snackbar>
+      <StandardBottomSheet snapPoints={["20%"]} hide={discardState.hide}>
+        <View className="px-[20px]">
+          <View className="relative flex items-center justify-center mt-[22px]">
+            <Heading3>{t( "discard-photos" )}</Heading3>
```

Comment:
This should use `Header4`, not `Header3`, according to the design specs in Figma.